Adding a bigger size images to Manual collection

OleksandrKhohkulia opened the pull request:
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after): 
Adding a custom class that makes bigger images in the manual collection


Test URLs:
- Before: https://main--governor--ociostateofnebraska.aem.live/
- After: https://main--governor--ociostateofnebraska.aem.live/

OleksandrKhohkulia committed:
Adding a bigger size images to Manual collection

acarnwath commented on blocks/manual-collection/manual-collection.css:
```diff
@@ -0,0 +1,5 @@
+@media (min-width: 40em) {
```
All CSS changes are made via SCSS. This change should be added in this file: https://github.com/OCIOStateOfNebraska/stateofnebraska-aem/blob/main/styles/ne-global/_manual-collection.scss

I'd also recommend leveraging a simpler class for authors so that they don't mis-type it. Perhaps just `Manual Collection (Bigger)`?

acarnwath replied:
@OleksandrKhohkulia please take another look at this link I provided - there are a couple blocks that take advantage of this class, so it is in a "global "space for better access. I think it would make sense to make this feature available to all of them.

OleksandrKhohkulia committed:
remove code from manual-collection.css

OleksandrKhohkulia committed:
manual collection pic size

Code moved to the scss file and class name changed to .bigger

OleksandrKhohkulia committed:
Update manual-collection.scss

OleksandrKhohkulia committed:
Update manual-collection.css

OleksandrKhohkulia committed:
"bigger" class added to _manual-collection.scss

Added a custom class called "bigger" to resize images to 15rem for screens larger than 40em within the manual-collection block.

kharris-reingold commented:
@OleksandrKhohkulia please take a look at the readme.md file in the root of the project. It has some details on the local development and build commands you'll want to use while working on this. The sass "scss" files are where styling changes occur but they need to be "compiled" down into css. This happens automatically when you run the 'npm run dev' command while doing local development. Once you've verified your changes in your local environment you would commit and push your changes (both scss and css) for PR review.

Looks like right now you only have your scss changes but not the compiled css.

OleksandrKhohkulia committed:
Update _manual-collection.scss
